src/main/haskell/kore/src/Kore/Step/Representation/OrOfExpandedPattern.hs

```diff
@@ -122,6 +123,26 @@ toExpandedPattern multiOr
             , substitution = mempty
             }
 
+{-| Transforms an 'OrOfExpandedPattern' into a 'StepPattern'.
+-}
+toStepPattern
+    ::  ( MetaOrObject level
+        , SortedVariable variable
+        , Ord (variable level)
+        , Show (variable level)
+        , Unparse (variable level)
+        )
+    => OrOfExpandedPattern level variable -> StepPattern level variable
+toStepPattern multiOr =
```

Comment:
Why not `ExpandedPattern.toMLPattern . toExpandedPattern`?

Reply:
If there are multiple patterns, won't that give two extra `\and`s?